mixer/pkg/runtime/config/ephemeral.go

```diff
@@ -204,6 +204,24 @@ func (e *Ephemeral) processHandlerConfigs(counters Counters, errs *multierror.Er
 	for key, resource := range e.entries {
 		var info *adapter.Info
 		var found bool
+
+		if key.Kind == constant.HandlerKind {
+			handlerProto := resource.Spec.(*config.Handler)
+			staticConfig := &HandlerStatic{
```

Comment:
I think instead of creating HandlerStatic, we should check if the `e.adapters[handlerProto.Adapter]` exists ? Because 
a. User might have it missed typed.
b. User might be intending to write a handler for the OOP adapter where they would write the handler stanza in the same way as for a compiled in adapter. so we should check if the `handlerProto.Adapter` does not exists inside `e.adapters` then just skip.